data/announcements/scheduled.yaml

```
@@ -116,3 +116,16 @@ announcements:
   message: |
     5 days of incredible opportunites to collaborate, learn + share with the entire community!<br />
     May 16 - 20, 2022.
+
+- name: Kubecon 2022 NA
+  startTime: 2022-10-15T00:00:00 #Added in https://github.com/kubernetes/website/pull/36107
```

Not sure why add this unnecessary comment. We can check git log for it.
```suggestion
  startTime: 2022-10-15T00:00:00 # Added in https://github.com/kubernetes/website/pull/36107
```

---

True but I imagined this to be a convention contributors follow when adding a new banner in Kubernetes.io
@sftim any thoughts?

---

The steering committee seem to like it (and they are the approvers for these changes).

```suggestion
  startTime: 2022-10-15T00:00:00 #Added in https://github.com/kubernetes/website/pull/36107 
```